packages/docusaurus-plugin-content-blog/src/__tests__/__snapshots__/feed.test.ts.snap

```diff
@@ -13,14 +13,14 @@ exports[`blogFeed atom shows feed item for each post 1`] = `
     <rights>Copyright</rights>
     <entry>
         <title type=\\"html\\"><![CDATA[MDX Blog Sample with require calls]]></title>
-        <id>MDX Blog Sample with require calls</id>
+        <id>/mdx-require-blog-post</id>
```

Comment:
Should you be wondering: the `id` field doesn't really matter for the feeds. [The example of the library itself](https://github.com/jpmonette/feed#example) uses `post.url` as the ID.